pkg/apis/acid.zalan.do/v1/postgresql_type.go

```diff
@@ -197,7 +197,8 @@ type CloneDescription struct {
 	S3Endpoint        string `json:"s3_endpoint,omitempty"`
 	S3AccessKeyId     string `json:"s3_access_key_id,omitempty"`
 	S3SecretAccessKey string `json:"s3_secret_access_key,omitempty"`
-	S3ForcePathStyle  *bool  `json:"s3_force_path_style,omitempty" defaults:"false"`
+	S3ForcePathStyle  *bool  `json:"s3_force_path_style,omitempty" defaults:"false"`	
+	TimelineID 		  string `json:"clone_target_timeline" defaults:"latest"`
```

Comment:
Please use go.fmt for right code formatting. This is misaligned.

Reply:
The type has been changed and the formatting is corrected.